src/main/java/com/example/bookshop/controller/BookController.java

```diff
@@ -45,8 +47,10 @@ public void delete(@PathVariable Long id) {
         bookService.deleteById(id);
     }
 
+    @ResponseStatus(HttpStatus.ACCEPTED)

```

Comment:
response status ACCEPTED is wrong here

ACCEPTED means that a request was accepted for processing, i.e. this processing will happen in the nearest future

this is not our case since update is happening right after request - therefore we should use 200 OK here

and since OK is the default response - you can remove `@ResponseStatus` at all